framework/src/model.cpp

```diff
@@ -456,7 +456,7 @@ namespace gvk
 
 				auto& weights = result.emplace_back(0.0f, 0.0f, 0.0f, 0.0f);
 				for (size_t j = 0; j < std::min(size_t{4}, vTempWeightsPerVertex[i].size()); ++j) {
-					weights[j] = std::get<float>(vTempWeightsPerVertex[i][j]);
+					weights[static_cast<int>(j)] = std::get<float>(vTempWeightsPerVertex[i][j]);
 				}
```

Comment:
Oh... wow, I never knew that GLM uses `int` as the length type by default. But that's actually cool.

However, with that knowledge, the optimal way of refactoring this code would be like follows:
```
const auto numWeights = std::min(int{4}, static_cast<int>(vTempWeightsPerVertex[i].size()));
for (int j = 0; j < numWeights; ++j) {
	weights[j] = std::get<float>(vTempWeightsPerVertex[i][j]);
}
```
We don't need the `size_t`.